frontend/packages/cli/src/cli/index.test.ts

```diff
@@ -21,6 +22,9 @@ describe('program', () => {
   it('should have the correct name and description', () => {
     expect(program.name()).toBe('liam')
     expect(program.description()).toBe('CLI tool for Liam')
+    const require = createRequire(import.meta.url)
+    const { version: packageVersion } = require('../../package.json')
+    expect(program.version()).toBe(packageVersion)
```

Comment:
Since this is a test, it would be preferable to use a literal value for `packageVersion`. However, considering that it is a value that gets updated frequently, I chose not to do so.